remote

```diff
@@ -297,7 +297,13 @@ function remote {
 			--output table \
 			--query "TrainingJobSummaries[:10].{JobName:TrainingJobName,Status:TrainingJobStatus}"
 
-    else
+	elif [[ "$1" == 'price' ]]; then
+		INSTANCE_TYPE=$2
+		curl -s --request GET \
+          	 --url https://api.vantage.sh/v1/products/aws-ec2-$INSTANCE_TYPE/prices/aws-ec2-$INSTANCE_TYPE-eu_west_1-on_demand-linux \
```

Comment:
Are we ok assuming we'll be on eu_west_1?

Reply:
We can add an issue to extend this in the future with an env variable, in Wellcome case I think we are mainly in `eu-west-1`

Reply:
An alternative is to return prices for multiple regions

Reply:
lets assume for now and extend later. btw sometimes people set the `AWS_REGION` env var when not working with aws profiles, so that's a not too bad way of picking it up.